ui/src/components/Publication/Creation/LinkedItems/index.tsx

```diff
@@ -113,34 +113,36 @@ const Links: React.FC = (): React.ReactElement => {
             <div className="relative">
                 <Components.PublicationCreationStepTitle text="Add links" required />
                 <div className="flex flex-col flex-wrap gap-4 sm:flex-row sm:items-center">
-                    <select
-                        name="linked-entity-type"
-                        id="linked-entity-type"
-                        onChange={(e) => {
-                            const value: Types.LinkedEntityType = e.target.value as Types.LinkedEntityType;
-                            setEntityType(value);
-                        }}
-                        value={entityType}
-                        className="block rounded-md border border-grey-200 outline-none focus:ring-2 focus:ring-yellow-500 sm:mr-0"
-                    >
-                        <option value="PUBLICATION">Publications</option>
-                        <option value="TOPIC">Research topics</option>
-                    </select>
+                    {!!isProblem && (
```

Comment:
"isProblem" is already a boolean. Does it need to be converted to a boolean again?

Reply:
No, there is no need